scripts/pi-hole/js/utils.js

```diff
@@ -104,8 +104,9 @@ function showAlert(type, icon, title, message) {
   }
 }
 
-function datetime(date) {
-  return moment.unix(Math.floor(date)).format("Y-MM-DD [<br class='hidden-lg'>]HH:mm:ss z");
+function datetime(date, html) {
+  var format = html === false ? "Y-MM-DD HH:mm:ss z" : "Y-MM-DD [<br class='hidden-lg'>]HH:mm:ss z";
```

Comment:
This works because `false !== undefined`, so if we don't pass `false` as the second parameter, the condition is false.